uplink/metainfo/kvmetainfo/buckets.go

```diff
@@ -60,7 +61,17 @@ func (db *Project) CreateBucket(ctx context.Context, bucketName string, info *st
 	}
 
 	info.Name = bucketName
-	newBucket, err := db.buckets.Create(ctx, *info)
+
+	// uuid MarshalJSON implementation always returns err == nil
+	partnerID, _ := info.PartnerID.MarshalJSON()
```

Comment:
Would `PartnerID` be nil?

Reply:
Its `UUID [16]byte` so I think it cannot be nil